Description: fix test hashes for Debian builds
 The tests pick up some data from the background build environment that differs
 from upstream.  This patch corrects the expected output on Debian.  So far,
 these hashes are stable across different builds andversions of EFL, as well as
 different minor releases of terminology.
 .
 See https://sourceforge.net/p/enlightenment/mailman/message/36634783/
Author: Ross Vandegrift <ross@kallisti.us>
Origin: other
Forwarded: no
Last-Update: 2019-05-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/tests.results
+++ b/tests/tests.results
@@ -77,7 +77,7 @@
 uts.sh 075ff9f8c4f718784330bac4fcae932b
 vpa.sh 4614f3c337e9d47fed3a4233df719617
 decswbv.sh f90ffe93b1bd99c6b7385ca71c6793de
-c2.sh 9f264ee84b6d89f1f9fa7b76abd442c0
+c2.sh c1d8061531d89476be61e948007ca239
 cell_is_empty.sh 507bc5dca92d6f0d4c081a5146c13ed6
 selection_simple.sh 1b3a915bf40ba8e869021055d368edf2
 selection_box_simple.sh 1b3a915bf40ba8e869021055d368edf2
@@ -120,5 +120,5 @@
 mouse_reporting_mode_normal_ext_urxvt.sh bf68099047410f102e71d2e451875c69
 mouse_reporting_mode_mouse_move_pressed_ext_urxvt.sh 59f3e59b60b3a43260de6a80135489eb
 mouse_reporting_mode_all_ext_urxvt.sh a11bbc8ebea42d06cc3ad823b52c09b3
-shift_in_out.sh 3896bcc6887998bbc41b43840a6b2e33
+shift_in_out.sh c3bb903237d9ef40d39ec07f328d922a
 osc-11-query.sh b762d503e40641ff896dac46391ad7a8
